src/components/Search/SearchByNameResults.tsx

```diff
@@ -0,0 +1,45 @@
+import { useTranslation } from 'react-i18next'
+import { UdtQueryResult } from '../../services/ExplorerService/fetcher'
+import styles from './SearchByNameResults.module.scss'
+import { truncateHash } from '../../utils/string'
+
+type Props = {
+  udtQueryResults: UdtQueryResult[]
+  truncateTypeHash?: boolean
+}
+
+export const SearchByNameResults = (props: Props) => {
+  const { udtQueryResults, truncateTypeHash } = props
+  return (
+    <div className={styles.searchResultsPanelWrapper}>
+      {udtQueryResults.length === 0 ? (
+        <EmptySearchByNameResult />
+      ) : (
+        udtQueryResults.map(item => {
+          return <SearchByNameResult key={item.typeHash} item={item} truncateTypeHash={truncateTypeHash} />
+        })
+      )}
+    </div>
+  )
+}
+
+const EmptySearchByNameResult = () => {
+  const { t } = useTranslation()
+  return <>{t('search.no_search_result')}</>
+}
+
+const SearchByNameResult = (props: { item: UdtQueryResult; truncateTypeHash?: boolean }) => {
+  const { t } = useTranslation()
+  const { item, truncateTypeHash } = props
+  const { typeHash, symbol } = item
+  const displayTypeHash = truncateTypeHash ? truncateHash(typeHash) : typeHash
+  return (
+    <>
+      <a className={styles.searchResult} href={`${window.origin}/sudt/${typeHash}`}>
+        {symbol ?? t('udt.unknown_token')}
+        <span className={styles.typeHash}>{displayTypeHash}</span>
+      </a>
+      <div className={styles.divider} />
```

Comment:
Why not use `border-bottom`?

Reply:
They have very similar appearance, while `div` has more flexibility when styling

Reply:
In this layout, I don't see the need for its flexibility, and it would be feasible to replace it with a more flexible HTML implementation in the future if needed.

Therefore, to avoid any misunderstandings (as this is an unconventional design), I suggest using the `border` implementation.